src/Imaging/GlideUrlBuilder.php

```diff
@@ -58,6 +58,10 @@ public function build($item, $params, $filename = null)
             $path .= Str::ensureLeft(URL::encode($filename), '/');
         }
 
-        return URL::prependSiteRoot($builder->getUrl($path, $params));
+        if (config('statamic.site.glide_prepend_siteroot')) {
```

Comment:
This is pointing to the wrong config value.

After changing the config location in my other comment, this should be `statamic.assets.image_manipulation.prepend_site_root`